accessories/NeoPixelLightStrip.py

```diff
@@ -0,0 +1,149 @@
+# An Accessory for Adafruit NeoPixels attached to GPIO Pin18
+# Tested using Python 3.6 Raspberry Pi
+# This device uses all available services for the Homekit Lightbulb API
+# Note: set your neopixels settings under the #NeoPixel constructor arguments
+# Note: RPi GPIO must be PWM. Neopixels.py will warn if wrong GPIO is used
+#       at runtime
+# Note: This Class requires the installation of rpi_ws281x lib
+#       Follow the instllation instructions;
+#           git clone https://github.com/jgarff/rpi_ws281x.git
+#           cd rpi_ws281x
+#           scons
+#
+#           cd python
+#           sudo python3.6 setup.py install
+# https://learn.adafruit.com/neopixels-on-raspberry-pi/software
+
+# Apple Homekit API Call Order
+# User changes light settings on iOS device
+# Changing Brightness - State - Hue - Brightness
+# Changing Color      - Saturation - Hue
+# Changing Temp/Sat   - Saturation - Hue
+# Changing State      - State
```

Comment:
Instead of using single line comments, maybe use python doc comments: `"""" """`